WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.swift

```diff
@@ -656,15 +654,6 @@ class AbstractPostListViewController: UIViewController,
         }
     }
 
-    private func didConfirmPublish(for post: AbstractPost) {
-        WPAnalytics.track(.postListPublishAction, withProperties: propertiesForAnalytics())
```

Comment:
I noticed we aren't tracking .postListPublishAction and .postListScheduleAction explicitly. We track it through

- `post_repository_post_created <post_type: post, status: publish>`
- `post_repository_post_updated <post_type: post, status: publish>`
- `post_repository_post_updated <post_type: post, status: future>`

If you think that's OK, that's fine with me - just wanted to note this.

Reply:
Great catch. I think it's a bug, and the correct place to track it is on the button tap. I'm going to add it directly to the button handler:

```
func publish(_ post: AbstractPost) {
        WPAnalytics.track(.postListPublishAction, withProperties: propertiesForAnalytics())
```

It't be great to generalize is since we already have an enum to represent these actions.